### PR TITLE
Fixed #31003 -- Doc'd and tested return value of QuerySet.bulk_create().

### PR DESCRIPTION
**Pull request by adamchainz**

### CODE REVIEW

File: docs/ref/models/querysets.txt [hunk 1]
```diff
@@ -2065,6 +2065,8 @@ are)::
     ...     Entry(headline='This is only a test'),
     ... ])
 
+Returns ``objs``.
```
Comment (pope1ni) [inline docs/ref/models/querysets.txt]: It is mentioned in the caveats below, but this is `objs` coerced to a `list`, and each object will only have the `pk` set if the query was against PostgreSQL.

I don't know whether we want to be clearer and state that it a modified version of `objs` in accordance with the caveats below?

Comment (adamchainz) [reply docs/ref/models/querysets.txt]: I moved it to the bottom, so it should be read after the caveats and description of the arguments, and added wording to indicate it's cast to a list.

**Comment by felixxm** [comment]
@adamchainz Thanks, I think it's a fix for ticket-31003. Can you add a note that `objs` are returned in the same order?

**Comment by felixxm** [comment]
@adamchainz Thanks 👍 We don't need an extra tests IMO. Ordering is already tested in multiple places in `postgres_tests`.

**Comment by adamchainz** [comment]
But what if Django drops PostgreSQL support in a future release?

Anyway I only added an assertion in an existing test, I think it should be fine.

**Comment by pope1ni** [comment]
I agree with Adam. The fact that order is maintained has nothing to do with PostgreSQL - adding a simple assertion here makes sense to me.

**Pull request merged**